### PR TITLE
fix yum install docs

### PR DESCRIPTION
**Pull request by nathwill**
cat sans sudo is no go

### CODE REVIEW

**Comment by nathwill** [comment]
um, ghost build failure?

**Comment by thaJeztah** [comment]
Thanks @nathwill, LGTM

ping @tianon could you have a quick look at this as well?

**Comment by LK4D4** [comment]
ping @tianon @jfrazelle

**Comment by jfrazelle** [comment]
LGTM

File: docs/installation/centos.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ package manager.
 
 3. Add the yum repo.
 
-        $ cat >/etc/yum.repos.d/docker.repo <<-EOF
+        $ echo | sudo tee /etc/yum.repos.d/docker.repo <<-EOF
```
Comment (phemmer) [inline docs/installation/centos.md]: Why the `echo |`? Serves no purpose that I can see as STDIN to the `sudo tee` comes from the heredoc, not the `echo`.

Comment (nathwill) [inline docs/installation/centos.md]: ha, sure enough. it fit the existing pattern and worked i guess. it does appear superfluous.

**Pull request merged**